quartz/components/scripts/search.inline.ts

```diff
@@ -248,6 +266,34 @@ document.addEventListener("nav", async (e: unknown) => {
     }
   }
 
+  async function fetchContent(slug: FullSlug) {
+    const targetUrl = new URL(resolveUrl(slug))
+    const contents = await fetch(targetUrl.toString())
+      .then((res) => res.text())
+      .catch((err) => {
+        console.error(err)
+      })
```

Comment:
early return on error here by checking if `contents === undefined`

Reply:
This is now addressed.